underfs/local/src/main/java/tachyon/underfs/local/LocalUnderFileSystemFactory.java

```diff
@@ -38,6 +38,8 @@ public boolean supportsPath(String path, TachyonConf tachyonConf) {
     if (path == null) {
       return false;
     }
-    return path.startsWith(TachyonURI.SEPARATOR) || path.startsWith("file://");
+    return path.startsWith(TachyonURI.SEPARATOR)
+        || path.startsWith("file://")
+        || TachyonURI.hasWindowsDrive(path, false);
```

Comment:
should this be
`|| (OSUtils.isWindows() && TachyonURI.hasWindowsDrive(path, false))`?